demos/android-supabase-todolist/app/build.gradle.kts

```diff
@@ -0,0 +1,97 @@
+import java.util.Properties
+
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.jetbrains.kotlin.android)
+    alias(libs.plugins.sqldelight)
+}
+
+val localProperties = Properties()
+val localPropertiesFile = rootProject.file("local.properties")
+if (localPropertiesFile.exists()) {
+    localPropertiesFile.inputStream().use { localProperties.load(it) }
+}
+
+fun getLocalProperty(key: String, defaultValue: String): String {
+    return localProperties.getProperty(key, defaultValue)
+}
+
+android {
+    namespace = "com.example.androidexample"
+    compileSdk = 34
+
+    buildFeatures {
+        buildConfig = true
+    }
+
+    defaultConfig {
+        applicationId = "com.example.androidexample"
```

Comment:
Is it okay to keep this as the default? It's a nit, but it might be nice to make this unique, maybe referencing `PowerSync`. In (rare) cases such as if dev's are using this default for their own (testing) apps this would cause the other app to be overwritten on install.

Reply:
Changed to use com.powersync instead